packages/astro/src/@types/astro.ts

```diff
@@ -750,8 +750,11 @@ export interface AstroUserConfig {
 	 * @type {boolean}
 	 * @default `true`
 	 * @description
-	 * This is an option to minify your HTML output and reduce the size of your HTML files. By default, Astro removes all whitespace from your HTML, including line breaks, from `.astro` components. This occurs both in development mode and in the final build.
-	 * To disable HTML compression, set the `compressHTML` flag to `false`.
+	 *
+	 * This is an option to minify your HTML output and reduce the size of your HTML files, occuring both in development mode and in the final build. 
+	 * By default, Astro removes whitespace from your HTML, including line breaks, from .astro components in a lossless manner.
+	 * It is important to note that some whitespace may be kept to preserve the visual rendering of your HTML.  
+	 * To disable HTML compression, set the compressHTML flag to false.
```

Comment:
```suggestion
	 * This is an option to minify your HTML output and reduce the size of your HTML files.
	 *
	 * By default, Astro removes whitespace from your HTML, including line breaks, from .astro components in a lossless manner. Some whitespace may be kept as needed to preserve the visual rendering of your HTML. This occurs both in development mode and in the final build.
	 *
	 * To disable HTML compression, set the compressHTML flag to false.
```
While we're here, a good chance to add some nicer formatting to an entry that hasn't been updated in a while! I always like separating out the definition at the top, and a statement like "To disable..." so they get their own paragraphs!